ml-proto/test/stdio_write.wast

```diff
@@ -0,0 +1,14 @@
+(module 
+  (import $write "stdio" "write" (param i32 i32))
+
+  (memory 4096 4096 (segment 0 "hello, world!\0D\0A\00"))
```

Comment:
```write``` is a base primitive for writing whatever to stdout. if you want to print text, your stdlib would do any transforms like ```\n``` -> ```\r\n``` on windows.